crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -1932,3 +1932,25 @@ async fn resolve_as_webpack_runtime(
 // TODO enable serialization
 #[turbo_tasks::value(transparent, serialization = "none")]
 pub struct AstPath(#[turbo_tasks(trace_ignore)] Vec<AstParentKind>);
+
+fn has_cjs_export(p: &Program) -> bool {
+    use swc_core::ecma::visit::{visit_obj_and_computed, Visit, VisitWith};
+
+    struct Visitor {
+        found: bool,
+    }
+
+    impl Visit for Visitor {
+        visit_obj_and_computed!();
+
+        fn visit_ident(&mut self, i: &Ident) {
+            if &*i.sym == "module" {
+                self.found = true;
```

Comment:
Is it possible to early exit a visitor? Babel has `path.stop()`.

Reply:
There's no way, but we can check for the flag in visitor methods.
I'll do it for `Stmt` and `Expr`. In my experience those were enough.